server/middleware/validation.js

```diff
@@ -155,6 +176,18 @@ const validators = {
     }
 };
 
+function getDay(date){
+    return date.split('-')[2];
+}
```

Comment:
Consider:
```suggestion
const getDay(date) => date.split('-')[2];
```

You're inconsistently using
```js
function foo(bar){
// and
function foo(bar) {
```

Use whichever's already used in this file.

Reply:
Good suggestion, updated.